### PR TITLE
Finished Registration and Added a feature to create new expenditure

### PR DESCRIPTION
**Pull request by Yaswanth-2874**

### CODE REVIEW

**Review by himasai-1423 — APPROVED** [review]

File: src/components/AddNewItem.jsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        balance: 2100,
+        isExpenditure: isExpenditure,
+      })
+      .then(() => alert("Success in insertion of data"))
+      .catch((e) => console.log(e));
+  };
+
+  return (
+    <form
+      style={{
```
Comment (himasai-1423) [inline src/components/AddNewItem.jsx:45]: Using CSS in a separate .css file will help in reusing the designs. Add a class and introduce a css file and using styling from that css file

File: src/pages/RegisterPage.jsx [hunk 2]
```diff
@@ -1,7 +1,95 @@
...
+    <form
+      style={{
+        display: "flex",
+        flexDirection: "column",
+        alignItems: "center",
+        gap: "20px",
+        padding: "20px 20px",
+        backgroundColor: "rgb(255, 255, 220)",
+        borderRadius: "30px",
+      }}
```
Comment (himasai-1423) [inline src/pages/RegisterPage.jsx:51]: Same here

**Review by himasai-1423 — CHANGES_REQUESTED** [review]
Hi @Yaswanth-2874 please address the comments. Thank you

File: src/pages/RegisterPage.jsx (hunk 2 above)
Comment (Yaswanth-2874) [reply src/pages/RegisterPage.jsx:51]: Not planning to change

**Review by himasai-1423 — APPROVED** [review]

**Pull request merged**